nexus-sdk/src/main/java/io/nexusrpc/handler/OperationMiddleware.java

```diff
@@ -0,0 +1,8 @@
+package io.nexusrpc.handler;
+
+/** Middleware for intercepting operations. */
+public interface OperationMiddleware {
+
+  /** Intercepts the given operation. Called once for each operation invocation. */
+  OperationHandler<Object, Object> intercept(OperationHandler<Object, Object> next);
```

Comment:
```suggestion
  OperationHandler<Object, Object> apply(OperationHandler<Object, Object> next);
```

I wonder if we don't need to call this "intercept" or these "interceptors" if we're adopting middleware-ish terminology. But I don't mind `intercept` either.

Reply:
Maybe `wrap`? But `intercept` also works for me.

Reply:
Left as `intercept` for now